.fixtures.yml

```diff
@@ -2,20 +2,20 @@ fixtures:
   forge_modules:
     stdlib:
       repo: "puppetlabs/stdlib"
-      ref: "6.0.0"
+      ref: "8.4.0"
```

Comment:
is there a reason for pinning? at vox pupuli we run rspec-puppet against the master branch of the dependencys. that enables us to catch breaking changes in deps before they are released. also we dont need to upgrade the fixtures file on a regular basis. for beaker tests, we run against the latest forge releases that the metadata.json allows.

Reply:
@bastelfreak I would agree but this isnt my project and i was copying the "status quo"

Reply:
@joshcooper what do you think? can we change all of these to main? it is best practice in most of the modules

Reply:
Assuming this file is only used in CI, I'd be fine changing to `main`